authors/apps/comments/commentReplyViews.py

```diff
@@ -1,12 +1,14 @@
 from rest_framework.generics import GenericAPIView, ListAPIView
 from rest_framework import status
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import ( IsAuthenticated,
+  IsAuthenticatedOrReadOnly )
 from rest_framework.response import Response
-from .serializers import ReplyLikeSerializer, ReplyCommentSerializer, ReplyHistorySerailizer
+from .serializers import (ReplyLikeSerializer, ReplyCommentSerializer,
+    ReplyHistorySerailizer )
 from authors.apps.profiles.models import Profile
 from authors.apps.articles.models import Article
 from authors.apps.authentication.models import User
-from .models import CommentReplyLike, Comment, CommentLike, CommentReply
```

Comment:
@MuhweziDeo Would please clarify as to why you removed the CommentLike model import statement?

Reply:
It was an unused import @calvinpete